### PR TITLE
[cleanup] Duplicate memory limit & Whitespace removed

### PR DESCRIPTION
**Pull request by valentin9**
- removed a duplicate memory_limit setting, now the latter setting is at the top, the latter is removed, as I thought 2G is more logical for composer installs & co in development.
- removed trailings space

### CODE REVIEW

**Comment by Neodork** [comment]
Other then the sendmail being hidden in the cleanup commit, LGTM!

File: cli/stubs/z-performance.ini [hunk 1]
```diff
@@ -16,13 +16,12 @@ post_max_size = 128M
...
 session.gc_probability = 0
 suhosin.session.cryptua = off
-memory_limit=2G
 
 ; Disable garbage collector
 zend.enable_gc = off
 
 ; Use mailhog for sending emails
-sendmail_path = /usr/local/bin/MailHog sendmail test@test  
+sendmail_path = /usr/local/bin/MailHog sendmail test@test
```
[the remark this answers is not in the document]
Comment (valentin9) [reply cli/stubs/z-performance.ini]: Hi!
What do you mean? It has a separate commit message "removed trailing whitespace".

Isn't it clear enough?

[the remark this answers is not in the document]
Comment (Neodork) [reply cli/stubs/z-performance.ini]: Right, my bad. You're correct I missed that while skimming over. Disregard my earlier comment.

**Comment by Neodork** [comment]
@samgranger LGTM review please.

**Comment by samgranger** [comment]
Looks good to me too! Merged.

**Pull request merged**